### DISCUSSION TITLE
Thank you

### DISCUSSION THREAD

**Discussion by zobbo** [tefkah/emacs-docs#5] [General]
Just a quick thanks for this, I'm finding it a lot more useful than trying to look within Emacs. Very much appreciated.

**Comment by tefkah** [comment]
Thank you, I really appreciate that! :)

**Comment by thaenalpha** [comment]
Thank you, now I've realized I'm not alone :)

**Comment by pevsner** [comment]
Thank you. 
An excellent resource.
Now my "go to" place for help.
